apps/studio/components/interfaces/Account/AccessTokens/ViewTokenPermissionsPanel.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  Sheet,
+  SheetContent,
+  SheetHeader,
+  SheetSection,
+  SheetFooter,
+  ScrollArea,
+  ScrollBar,
+  cn,
+  Badge,
+  Button,
+  Separator,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from 'ui'
+import { AccessToken } from 'data/access-tokens/access-tokens-query'
+import { DocsButton } from 'components/ui/DocsButton'
+import { Card, CardContent } from 'ui'
+import { ACCESS_TOKEN_PERMISSIONS } from './AccessToken.constants'
+import ConfirmationModal from 'ui-patterns/Dialogs/ConfirmationModal'
+import { useState } from 'react'
+import { Info } from 'lucide-react'
+
+interface ViewTokenPermissionsPanelProps {
+  visible: boolean
+  token: AccessToken | undefined
+  onClose: () => void
+  onDeleteToken?: () => void
+}
+
+export function ViewTokenPermissionsPanel({
+  visible,
+  token,
+  onClose,
+  onDeleteToken,
+}: ViewTokenPermissionsPanelProps) {
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false)
+
+  // Dummy access settings for demonstration
+  const getDummyAccess = (resource: string) => {
```

Comment:
This will be removed thankfully, once we have those permissions coming from somewhere.